### PR TITLE
Gj feature locker input

### PR DESCRIPTION
**Pull request by jingavin**
Locker input styled to design specifications

### CODE REVIEW

File: client/js/components/HalfAddPhoto/HalfAddPhoto.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (siddharthparmar7) [inline client/js/components/HalfAddPhoto/HalfAddPhoto.js]: Didn't you remove this component?

File: client/js/components/MapView/MapView.js [hunk 2]
```diff
@@ -80,7 +80,8 @@ class MapViewComponent extends Component {
                           latitude: e.nativeEvent.coordinate.latitude,
                           longitude: e.nativeEvent.coordinate.longitude
                         }
-                      });
+                      }),
+                        this.props.navigation.navigate("AddLocker", {});
```
Comment (siddharthparmar7) [inline client/js/components/MapView/MapView.js]: why are you passing an empty object with this navigation?

File: client/js/components/WideAddPhoto/WideAddPhoto.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
```
Comment (siddharthparmar7) [inline client/js/components/WideAddPhoto/WideAddPhoto.js]: Did you remove this component?

File: client/js/screens/AddLocker/AddLocker.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    } else if (response.error) {
+      return response.error;
+    } else {
+      updateFilesToUpload(response.uri);
+    }
+  });
+};
+
+onSubmit = () => {
+  console.log("submit submit");
```
Comment (siddharthparmar7) [inline client/js/screens/AddLocker/AddLocker.js]: remove this console log.

File: server/src/locker.js [hunk 5]
```diff
@@ -36,18 +36,18 @@ const csvToJSONObject = async () => {
...
       }
     });
+    return { data: await Promise.all(spots) };
   } catch (error) {
     return error;
   }
 };
 
-csvToJSONObject();
+// csvToJSONObject();
```
Comment (siddharthparmar7) [inline server/src/locker.js]: remove this commented code.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes.

File: client/js/components/HalfAddPhoto/HalfAddPhoto.js (hunk 1 above)
Comment (jingavin) [reply client/js/components/HalfAddPhoto/HalfAddPhoto.js]: removed unused component.

File: client/js/components/WideAddPhoto/WideAddPhoto.js (hunk 3 above)
Comment (jingavin) [reply client/js/components/WideAddPhoto/WideAddPhoto.js]: removed unused component.

File: client/js/screens/AddLocker/AddLocker.js (hunk 4 above)
Comment (jingavin) [reply client/js/screens/AddLocker/AddLocker.js]: removed console log.

File: server/src/locker.js (hunk 5 above)
Comment (jingavin) [reply server/src/locker.js]: Removed comment

File: client/js/components/MapView/MapView.js (hunk 2 above)
Comment (jingavin) [reply client/js/components/MapView/MapView.js]: Now passing lats and longs.

File: server/graphcool.yml [hunk 6]
```diff
@@ -37,8 +30,13 @@ permissions:
...
 
 # added by email-password template: (please uncomment)
-
 functions:
+  getLockers:
+    type: resolver
+    schema: types.graphql
+    handler:
+      code: src/locker.js
+
```
Comment (siddharthparmar7) [inline server/graphcool.yml]: Do you still need this?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**